### PR TITLE
Ability to add movie pricing

### LINKED ISSUE #40: FE Ability to add movie pricing
## Scope

Add pricing to existing movies once rights holder has been approved ~and chain of title is approved~ (not for now)

- Form to capture region(dropdown of UN_M49 tags), amount, from, to, movieMedium dropdown(  THEATER, EST, DTR,PAYTV,SVOD,FREETV)
- When RH visits movie details page a table of available movie prices is shown at bottom
- If movie does not have default pricing record display error message to RH that default price is required
- If movie does not have active prices i.e toWindow date has passed notify RH to add new price or change toWindow for existing prices
- Database amounts stored in wei but displayed to rights holder in DAI
- Region should store M49 code not ISO format.

## Gain
Ability to capture movie pricing

## Roles
gardener: @mwaa 50 DAI
worker: @mwaa 420 DAI
reviewer:  80 DAI

### PR DESCRIPTION
**Pull request by mwaa**
Please remember the [Contributing Guidelines](https://WhiteRabbitTheWorldIsYours/dashboard/blob/master/CONTRIBUTION.md) :heart:

## What does this PR do?
Add pricing forms for create, edit and delete

## Recommendations for testing
- [x] Using approved user visit  `/movie/prices/:IMDB`
- [x] You can also use `Prices` button found on list of movies belonging to RH
- [x] Use Add Price button to display the form. Save adds new entry, cancel hides form
- [x] If movie has pricing records you can click on edit and update existing record
- [x] Forms dismiss on success
- [x] Database stores amount in Wei but user sees xdai normal value 

## Links to relevant issues or information
Closes #40

### CODE REVIEW

**Comment by troggy** [comment]
I get the error if I leave "window" empty
image

**Comment by troggy** [comment]
Overall, there is an error if I click "Save" with incomplete form (e.g. with totally empty form)

**Comment by troggy** [comment]
Date format should be human-friendly
image

**Comment by troggy** [comment]
Also, IMO we don't need time for "window" parameter. I can't imagine a use case for time

**Comment by troggy** [comment]
-[ ] ✨

image

**Comment by troggy** [comment]
✅  checked with WR app (after hardcoding proper id and fixing backend url)

File: frontend/src/stores/UNM49.ts [hunk 1]
```diff
@@ -0,0 +1,251 @@
+export default {
+  Afghanistan: { M49: '004', ISO: 'AFG' },
```
Comment (Clearwood) [inline frontend/src/stores/UNM49.ts:2]: what are we using the ISO code for?

File: frontend/src/containers/components/PriceForm.tsx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+interface ComponentProps {
+  price: PriceInterface,
+  onClear: Function
+}
+
+const regionOptions = [<Select.Option value="001" key="001">DEFAULT</Select.Option>];
+const countries = Object.keys(Regions);
+countries.forEach((country) => {
+  if (Object.prototype.hasOwnProperty.call(Regions, country)) {
```
Comment (troggy) [inline frontend/src/containers/components/PriceForm.tsx:27]: why not `if (Regions[country]) {` ? 🤔

File: frontend/src/containers/MoviePrice.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+  const openPriceForm = (record: PriceInterface) => {
+    setCurrentPrice({ ...record, IMDB });
+  };
+
+  const columns = [
+    {
+      title: 'Region',
+      dataIndex: 'region',
+      key: 'region',
```
Comment (troggy) [inline frontend/src/containers/MoviePrice.tsx:37]: I suggest to show human-readable name in Region column (e.g. "Germany")

**Comment by Clearwood** [comment]
image
maybe the distance between new price and the table could be larger

**Comment by Clearwood** [comment]
image
Sites to change the price are viewable by any user, regardless if they are the rightsholder for this movie or not

File: frontend/src/stores/UNM49.ts (hunk 1 above)
Comment (troggy) [reply frontend/src/stores/UNM49.ts:2]: nothing I guess :)

**Comment by troggy** [comment]
added "Delete" button as well

**Comment by troggy** [comment]
addressed all the concerns except code style which is minor

File: frontend/src/stores/humanizeM49.ts [hunk 4]
```diff
@@ -0,0 +1,251 @@
+export default {
```
Comment (Clearwood) [inline frontend/src/stores/humanizeM49.ts:1]: Inconsistent format in JSON keys.

**Pull request merged**